training/templates/base.html

```diff
@@ -1,40 +1,128 @@
+	<!-- <nav class="navbar fixed-top navbar-dark navbar-expand-sm" id="masthead"> -->
+		<a class="navbar-brand mr-5" href="/tiaas">
+			<span class="navbar-brand-title">{{ settings.TIAAS_OWNER }} - Training Infrastructure as a Service</span>
+		</a>
+
+		<ul class="navbar-nav">
+			<li class="nav-item">
+				<a
+					class="nav-link nav-icon"
+					href="/"
+					data-toggle="tooltip"
+					data-placement="bottom"
+					data-title="Back to {{ settings.TIAAS_OWNER }}"
+				>
+					<i class="fa fa-fw fa-home"></i>
+				</a>
+			</li>
+
+			<li class="nav-item">
+				<a class="nav-link" href="/tiaas" id="nav-link-about">Home</a>
+			</li>
+
+			<li class="nav-item">
+				<a class="nav-link" href="/tiaas/calendar" id="nav-link-calendar">Calendar</a>
+			</li>
+
+			<li class="nav-item">
+				<a class="nav-link" href="/tiaas/stats" id="nav-link-stats">Stats</a>
+			</li>
+
+			<li class="nav-item">
+				<a class="nav-link" href="/tiaas/new" id="nav-link-request">Request</a>
+			</li>
+
+			{% if user.is_authenticated %}
+			<li class="nav-item">
+				<a class="nav-link" href="/tiaas/admin">Admin</a>
+			</li>
+			{% endif %}
+		</ul>
+
+		<div>
+			{% if user.is_authenticated %}
+			<a class="navbar-brand" href="/tiaas/admin/logout/">Logout</a>
+			{% endif %}
+		</div>
+	</nav>
+
+	<main>
+
+		{% block content %}
+		{% endblock %}
+
+	</main>
+
+	<footer>
+
+		{% include 'snippets/footer.html' %}
+
+		<div class="version">
+			<b>Training Infrastructure as a Service (TIaaS)</b>
+			licensed under the AGPLv3, running commit
+			<a
+				href="{{ settings.GIT_REMOTE_URL }}/tree/{{ settings.GIT_COMMIT_ID }}"
+				target="_blank"
+			>{{ settings.GIT_COMMIT_ID }}</a>
+		</div>
+	</footer>
+
+	<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/popper.js@1.12.9/dist/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
+	<script type="text/javascript">
+		// Set the navbar active link based on the current URL
+		const LOCATIONS = {
+			'tiaas': 'about',
+			'tiaas/new': 'request',
+			'tiaas/calendar': 'calendar',
+			'tiaas/stats': 'stats',
+		}
+		$(document).ready( () => {
+			// Set current page nav link to 'active'
+			const loc = window.location.pathname.replace(/^\/+|\/+$/g, '');
+			if (LOCATIONS[loc]) {
+				$('.nav-link').removeClass('active');
+				$(`#nav-link-${LOCATIONS[loc]}`).addClass('active');
+			}
+		})
+		// Enable bootstrap tooltips
+		$(function () {
+		  $('[data-toggle="tooltip"]').tooltip()
+		})
+	</script>
```

Comment:
This should not be necessary with Django, we know which page is being visited. Can we do this in a JS free way perhaps?

Reply:
Done

```html
<a class="nav-link{% if request.path == '/tiaas/' %} active{% endif %}" href="/tiaas" id="nav-link-about">Home</a>
```

Reply:
perf, cheers